libs/cpp/CMakeLists.txt

```diff
@@ -6,7 +6,7 @@ project(libfaasm)
 set(CMAKE_CXX_STANDARD 17)
 
 set(PROJ_INCLUDE ${CMAKE_CURRENT_LIST_DIR}/../../include)
-include_directories(${CMAKE_CURRENT_LIST_DIR} ${PROJ_INCLUDE})
+include_directories(${CMAKE_CURRENT_LIST_DIR} ${PROJ_INCLUDE} /usr/local/include)
```

Comment:
I don't feel good about this, it has the potential to introduce some gnarly issues depending on the build environment. What's the error caused without this?

Reply:
Those lines in `matrix.h`:
```cpp
#include <eigen3/Eigen/Core>
#include <eigen3/Eigen/SparseCore>
```
I agree it shouldn't be in there and should be deleted in the future, I'm happy with having it only locally instead

Reply:
Have you run `inv libs.eigen`? If you run `inv libs.toolchain` it should rebuild everything you need for a new toolchain. Think this is in the toolchain docs.

Reply:
Oh I remember now running into this before and needing to run `inv libs.eigen` thanks that fixes it :+1:.

I want to add a warning for me for `inv libs.faasm` because I treat it too often like it is `inv libs.toolchain`!